test/check-application

```diff
@@ -144,6 +144,27 @@ class TestApplication(testlib.MachineCase):
         self.execute(False, "podman run -d --name test-sh-user alpine sh")
         self.execute(False, "podman run -d --name swamped-crate-user busybox sleep 1000")
 
+        # Test owner filtering
+        if auth:
+            b.wait_js_func("ph_count_check", "#containers-images tbody", 8)
+            b.wait_js_func("ph_count_check", "#containers-containers tbody", 2)
+
+            b.set_val("#containers-containers-owner", "system")
+            b.wait_js_func("ph_count_check", "#containers-images tbody", 5)
+            b.wait_in_text("#containers-images", "system")
+            b.wait_js_func("ph_count_check", "#containers-containers tbody", 1)
+            b.wait_in_text("#containers-containers", "system")
+
+            b.set_val("#containers-containers-owner", "user")
+            b.wait_js_func("ph_count_check", "#containers-images tbody", 3)
+            b.wait_in_text("#containers-images", "admin")
+            b.wait_js_func("ph_count_check", "#containers-containers tbody", 1)
+            b.wait_in_text("#containers-containers", "admin")
+
+            b.set_val("#containers-containers-owner", "all")
```

Comment:
This could check that this gets back to 8 images and 2 containers.

Reply:
Done